config/samples/cluster_v1alpha1_cluster.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: cluster.k8s.io/v1alpha2
+apiVersion: cluster.k8s.io/v1alpha1
```

Comment:
The apiVersion would need to be updated with the new api group `cluster.sigs.k8s.io`.

That said, since these are not actually valid examples, it might be best to just remove these files. @ncdc thoughts?

Reply:
@detiber Should I make the changes?

Reply:
I would remove all of these sample files as they aren't valid. @vincepri is this something that kubebuilder generated?

Reply:
Yes, they are generated by kubebuilder.

Reply:
But it should be safe to remove them, right?

Reply:
Indeed, and it would cause less new developer confusion

Reply:
Great, let's proceed with their removal!

Reply:
@ncdc @detiber You intend to remove the entire samples folder or just the v1alpha1 samples?

Reply:
All of the samples because they are all autogenerated and invalid configs.

Reply:
Great, I will update.

Reply:
Can we add the folder to `.gitignore` after removing the files?

Reply:
@vincepri Can't we remove the folder instead?

Reply:
Remove the folder + add to `.gitignore` so when we use kubebuilder doesn't get readded :)

Reply:
Sure I am updating it.